docs/configuration/index.md

```diff
@@ -828,6 +828,40 @@ The information about releases [is retrieved from Bodhi](https://bodhi.fedorapro
 cache and required availability on Copr, it might take a while to get the
 newest state.
 
+### EPEL minor version aliases
+
+Starting with EPEL 10, there are [different dist-git branches](https://docs.fedoraproject.org/en-US/epel/branches/#_epel_10)
+and Koji targets for currently active minor versions. The main/default branch/target
+called `epelX` (e.g. `epel10`) that represents the latest (leading) minor release
+and corresponds to CentOS Stream X and the `epelX.Y` (e.g. `epel10.0`) branch/target
+that represents the previous (trailing) minor release and corresponds to RHEL X.Y.
```

Comment:
```suggestion
and Koji targets for currently active minor versions. 

The main/default branch/target called `epelX` (e.g. `epel10`)  represents the latest (leading) minor release
and corresponds to CentOS Stream X. 

The `epelX.Y` (e.g. `epel10.0`) branch/target represents the previous (trailing) minor release and corresponds to RHEL X.Y.
```

This is though to read. I would try to spilt it up.

Reply:
What I could do is to convert the second sentence into a 2-item list, that would help readability, but at the same time it would feel a bit weird to me.

Reply:
I was lost after the first `that` sentence, but maybe it is just me. Feel free to ignore my suggestion. Having a list can surely improve the readability.